### PR TITLE
13758 start moving away from shared server

### LINKED ISSUE #13758: Start converting servlet.4.0_fat away from SharedServer
Remove SharedServer references from:

WCGetMappingSlashStarTest
WCServletPathForDefaultMappingFalse
WCServletPathForDefaultMappingDefault

In addition moving WCServletContainerInitializerExceptionTest to a unique server to avoid having tests after it runs having to expect the error it produces when stopping the server.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #13758

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/server.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<server description="Server for the SCI Exception test.">
+
+  <include location="../fatTestPorts.xml"/>
+
+  <featureManager>
+    <feature>servlet-4.0</feature>
+  </featureManager>
+
+  <logging
+    traceSpecification="*=info=enabled:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HttpTransport=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all:com.ibm.ws.session.*=all:com.ibm.ws.runtime.update.internal*=all:ChannelFramework=all"
```
Comment (pmd1nh) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/server.xml:10]: Need the extra trace for JSP, runtime?

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+bootstrap.include=../testports.properties
+osgi.console=7777
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HttpTransport=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all:com.ibm.ws.session.*=all:com.ibm.ws.runtime.update.internal*=all:ChannelFramework=all
```
Comment (pmd1nh) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties:3]: extra traces needed?

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/server.xml (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/server.xml]: limited to WC traces

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties (hunk 2 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties]: limited to WC traces

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**